src/TesterInternal/General/BuiltInSerializerTests.cs

```diff
@@ -29,7 +29,7 @@ public class BuiltInSerializerTests
         public void InitializeForTesting()
         {
             TraceLogger.Initialize(new NodeConfiguration());
-            SerializationManager.Initialize(false, null);
+            SerializationManager.Initialize(false, null, false);
```

Comment:
Ohhh, I see now. It is because of the boxing. And later on Json is confused due to boxing and deserializes the boxed numerical value to long. And boxed long can not be directly converted to int. It needs to be unboxed first. Makes sense.

The problem of course is the boxing. I will see if I can change this API to return T[] and not object[]. That would be the correct solution, right?